### PR TITLE
@craigspaeth: Removes browser-sync

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: src/data/web.json [hunk 1]
```diff
@@ -172,8 +172,8 @@
     "bringing-together": {
       "headline": "2014 brought us one step closer to making art accessible to anyone with an internet connection.",
       "backdrop": {
-        "src": "richard-serra.jpg",
-        "credit": "Installation of Richard Serra’s <em>Backdoor Pipeline</em> (2010) at Gagosian Gallery, Britannia Street, London.<br>Photo by Mike Bruce."
+        "src": "eugene-lamay.jpg",
+        "credit": "Eugene Lemay, <em>Nights in Beirut</em>, 2014, courtesy of Mana Contemporary. Photo by E. Lee Smith."
       }
```
Comment (dzucconi) [inline src/data/web.json]: Guess I never merged this :p

**Comment by dblock** [comment]
:+1:

**Comment by craigspaeth** [comment]
:+1: for the illustration!

**Comment by craigspaeth** [comment]
And :100: for helping out with this!

File: src/data/web.json (hunk 1 above)
Comment (mzikherman) [inline src/data/web.json:177]: lol

**Comment by pwz3n0** [comment]
:see_no_evil:
:hear_no_evil:
:speak_no_evil:

**Pull request merged**